askama_parser/benches/from_str/librustdoc/LICENSE.md

```diff
@@ -0,0 +1,182 @@
+The files in this folder were copied verbatim from [librustdoc/html/templates] in version
+eeb59f16a5f40e14dc29b95155b7f2569329e3ec. Dual licensed under MIT OR Apache-2.0.
+
+Please find the authors in [their Git history].
+
+[librustdoc/html/templates]: <https://github.com/rust-lang/rust/tree/master/src/librustdoc/html/templates>
+[their Git history]: <https://github.com/rust-lang/rust/commits/master/src/librustdoc/html/templates>
+
+---
```

Comment:
Let's skip the shortlog, I don't think we need this.

Also until we actually have other sources of benchmarks in here, I'd prefer to skip one level of directory hierarchy here, so we should have `benches/main.rs` and `benches/librustdoc/*`.

Reply:
Removed.